tabs/mixer.js

```diff
@@ -333,22 +333,51 @@ TABS.mixer.initialize = function (callback, scrollPosition) {
             content: $('#mixerApplyContent')
         });
 
-        $('#execute-button').click(function () {
-            helper.mixer.loadServoRules(currentMixerPreset);
-            helper.mixer.loadMotorRules(currentMixerPreset);
+        heliModal = new jBox('Modal', {
+            width: 480,
+            height: 240,
+            closeButton: 'title',
+            animation: false,
+            title: chrome.i18n.getMessage("heliModalTitle"),
+            content: $('#heliModalContent')
+        });
+
+        function mixerApply() {
+            heliModal.close();
```

Comment:
this is not enough. jBox we have over here has a bug that causes events to be rebinded every time modal is opened. We had some troubles with it before previous release. If user opens modal, closes, and then opens it again, there will be 2 events binded to modal elements in total.
The workaround is to remove modal wrapper and delete the modal instance as well. See ae6c37118d1da1ce1ea360cd67c4b078f6ec62f4#diff-7db851820cf7a896309fa2ecbee7e185

```
delete OSD.GUI.jbox;
$('.jBox-wrapper').remove()
```

Reply:
Understood

Reply:
Fixed